prime-angular-frontend/src/app/modules/adjudication/shared/components/admin-users-table/admin-users-table.component.html

```diff
@@ -0,0 +1,68 @@
+<div class="row">
+  <div class="col">
+    <table mat-table
+           class="w-100"
+           [dataSource]="dataSource">
+
+      <ng-container matColumnDef="username">
+        <th mat-header-cell
+            *matHeaderCellDef> Username
+        </th>
+        <td mat-cell
+            *matCellDef="let row;"> {{ row.username }} </td>
+      </ng-container>
+
+      <ng-container matColumnDef="firstname">
+        <th mat-header-cell
+            *matHeaderCellDef> Firstname
+        </th>
+        <td mat-cell
+            *matCellDef="let row;"> {{ row.firstName }} </td>
+      </ng-container>
+
+      <ng-container matColumnDef="lastname">
+        <th mat-header-cell
+            *matHeaderCellDef> Lastname
+        </th>
+        <td mat-cell
+            *matCellDef="let row;"> {{ row.lastName }} </td>
+      </ng-container>
+
+      <ng-container matColumnDef="siteassignment">
+        <th mat-header-cell
+            *matHeaderCellDef> Site Assignment
+        </th>
+        <td mat-cell
+            *matCellDef="let row;"> {{ row.siteAssignment }} </td>
+      </ng-container>
+
+      <ng-container matColumnDef="enrolleeassignment">
+        <th mat-header-cell
+            *matHeaderCellDef> Enrollee Assignment
```

Comment:
I think it would be more clear if renamed to "Enrollees Assigned".  Can you please update class field names as well?

Reply:
Updated as suggested

Reply:
@bergomi02 please make it "Enrollees Assigned", with a "s"